source/Core/Mangled.cpp

```diff
@@ -49,13 +35,14 @@
 #include <ctype.h>
 #include <functional>
 #include <mutex>
-#include "lldb/lldb-enumerations.h" // for LanguageType
+#include "lldb/lldb-enumerations.h"
 
 #include "Plugins/Language/CPlusPlus/CPlusPlusLanguage.h"
 #include "Plugins/Language/ObjC/ObjCLanguage.h"
 
-#include "llvm/ADT/StringRef.h"    // for StringRef
-#include "llvm/Support/Compiler.h" // for LLVM_PRETT...
+#include "llvm/ADT/StringRef.h"
+#include "llvm/Demangle/Demangle.h"
+#include "llvm/Support/Compiler.h"
 
```

Comment:
Had a merge conflict in the lines above, because I added `lldb/Core/RichManglingContext.h` and removed comments on includes.